tests/drivers/dma/chan_blen_transfer/boards/s32z2xxdc2_s32z270_rtu1.overlay

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/memory-attr/memory-attr-arm.h>
+
+/ {
+	soc {
+		sram1: memory@35780000 {
+			compatible = "mmio-sram";
+			reg = <0x31780000 DT_SIZE_K(960)>;
+		};
+
+		sram2: memory@35870000 {
+			compatible = "zephyr,memory-region", "mmio-sram";
+			reg = <0x35870000 DT_SIZE_K(64)>;
+			zephyr,memory-region = "SRAM3";
```

Comment:
`SRAM2`? or perhaps use `sram_nocache` for node lable and `SRAM_NOCACHE` for `zephyr,memory-region`. Use the same approach for all places

Reply:
I updated

Reply:
why not `sram2`?

Reply:
I updated `sram_nocache` for node lable and `SRAMNOCACHE` for `zephyr,memory-region`.